pkg/registry/Makefile.am

```diff
@@ -11,7 +11,9 @@ registry_VIEWS = \
 	pkg/registry/views/imagestream-meta.html \
 	pkg/registry/views/imagestream-page.html \
 	pkg/registry/views/imagestream-panel.html \
-	pkg/registry/views/project-new-dialog.html \
+	pkg/registry/views/project/add-group-dialog.html \
```

Comment:
Because we already have nested directories for the different types of files. Much like the openshift project. If we had chosen to put all the project stuff in one directory (scripts, views, styles) then this would make sense, but we've chosen to have directories by asset type.